tests/Constraints/RedirectsTo.php

```diff
@@ -19,7 +19,7 @@ class RedirectsTo extends Constraint
      * @param int $redirects
      * @return void
      */
-    public function __construct($destination, int $redirects)
+    public function __construct($destination, int $redirects = 1)
```

Comment:
This serves no purpose, the constructor is never called directly

Reply:
I added this for consistency. If the assert method has this as a default value, then the constructor also should have this as the default value.

Maybe this one comes to believes so if it bothers you I can revert it.